playwright.config.ts

```diff
@@ -0,0 +1,89 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './playwright/e2e/suites',
+  // Folder for test artifacts such as screenshots, videos, traces, etc.
+  outputDir: './playwright/test-results',
```

Comment:
If you ran playwright from an inner directory, would it create these directories there instead of at the root directory?
If so, using `${__dirname}/playwright/e2e/suites` and `${__dirname}/playwright/test-results` may help get the absolute path - otherwise no issue.

Reply:
Nope, the location is consistent